src/main/java/com/aws/greengrass/deployment/errorcode/DeploymentErrorCode.java

```diff
@@ -94,6 +94,7 @@ public enum DeploymentErrorCode {
     DOCKER_LOGIN_ERROR(DeploymentErrorType.DEPENDENCY_ERROR),
     DOCKER_PULL_ERROR(DeploymentErrorType.DEPENDENCY_ERROR),
     DOCKER_IMAGE_NOT_VALID(DeploymentErrorType.DEPENDENCY_ERROR),
+    DOCKER_RMI_ERROR(DeploymentErrorType.DEPENDENCY_ERROR),
```

Comment:
you don't need this error code because failure to remove docker image should not fail a deployment